classes/class-wc-connect-taxjar-integration.php

```diff
@@ -27,6 +27,33 @@ public function init() {
 	}
 
 	/**
+	 * Gets the store's location settings.
+	 *
+	 * Modified version of TaxJar's plugin.
+	 * See: https://github.com/taxjar/taxjar-woocommerce-plugin/blob/82bf7c58/includes/class-wc-taxjar-integration.php#L796
+	 *
+	 * @return array
+	 */
+	public function get_store_settings() {
+		$default_wc_settings     = explode( ':', get_option( 'woocommerce_default_country' ) );
+		$taxjar_city_setting     = get_option( 'woocommerce_store_city' );
+		$taxjar_zip_code_setting = get_option( 'woocommerce_store_postcode' );
```

Comment:
Should we use the `WC_Countries` functions instead? (if they exist). They filter the option values.